sql/core/src/test/scala/org/apache/spark/sql/LegacyErrorTempSuit.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Created by ruilibuaa
+ * 2023-3-19
+ */
+
+package org.apache.spark.sql
+
+import org.apache.spark.SparkException
+import org.apache.spark.sql.expressions.UserDefinedFunction
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.test.SharedSparkSession
+
+
+class LegacyErrorTempSuit extends SharedSparkSession {
+
+
+  test("CANNOT_ZIP_MAPS") {
```

Comment:
Could you place the test to `QueryCompilationErrorsSuite`, please. Usually, we put tests for exceptions from:
- QueryParsingErrors to QueryParsingErrorsSuite
- QueryCompilationErrors to QueryCompilationErrorsSuite
- QueryExecutionErrors to QueryExecutionErrorsSuite